independent-publisher-connectors/ClickUp Team Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1231 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "ClickUp Team Manager",
+    "description": "Connects to the ClickUp team to manage tasks, folders, and lists from within Power Platform",
+	    "contact": {
+      "name": "Duke DeVan",
+      "url": "https://dukedevan.com",
+      "email": "ddevan@hitachisolutions.com"
+    }
+  },
+  "host": "api.clickup.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/v2/space/{space_id}/folder": {
+      "post": {
+        "summary": "Create a Folder",
+        "description": "Creates a new folder in your space",
+        "operationId": "CreateAFolder",
+        "parameters": [
+          {
+            "name": "space_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+			"x-ms-summary":"Create folder new folder",

```

Comment:
@marsurice  Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer the certified connectors in the open-source repo for reference.